debugging.md

```diff
@@ -22,6 +22,22 @@ issues.
 
 ### Missing RLS component
 
+### stable, beta toolchains
+
+You might see an error like `toolchain 'stable-x86_64-unknown-linux-gnu' does not contain component 'rls' for target 'x86_64-unknown-linux-gnu'`.
```

Comment:
I imagine you could say: '(...), however we guarantee for stable and beta toolchains to contain the `rls` component'` instead of the next line